### PR TITLE
[DUOS-885] Table/Visual Ease & Accessibility

### PR DESCRIPTION
**Pull request by solideoglori**
**ADDRESSES**
https://broadinstitute.atlassian.net/browse/DUOS-885

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] PR is labeled with a Jira ticket number and includes a link to the ticket
- [ ] PR is labeled with a security risk modifier [no, low, medium, high] 
- [ ] PR describes scope of changes

In all cases:

- [ ] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [ ] Get PO sign-off for all non-trivial UI or workflow changes
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/DatasetCatalog.js [hunk 1]
```diff
@@ -392,23 +393,23 @@ class DatasetCatalog extends Component {
...
-                    th({ className: 'table-titles dataset-color cell-size' }, ['# of Participants']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Description']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Species']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Data Custodian']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['Consent ID']),
-                    th({ className: 'table-titles dataset-color cell-size' }, ['SC-ID']),
-                    th({ isRendered: this.state.isAdmin, className: 'table-titles dataset-color cell-size' }, ['Approved Requestors'])
+                    th({ className: 'cell-size' }, ['Dataset ID']),
+                    th({ className: 'cell-size' }, ['Dataset Name']),
+                    th({ className: 'cell-size'}, ['Data Access Committee']),
```
Comment (rushtong) [inline src/pages/DatasetCatalog.js]: super minor spacing nit: 
```suggestion
                    th({ className: 'cell-size' }, ['Data Access Committee']),
```

File: src/pages/DatasetCatalog.js [hunk 2]
```diff
@@ -615,7 +616,7 @@ class DatasetCatalog extends Component {
               onClick: this.download,
               className: 'col-lg-5 col-md-5 col-sm-5 col-xs-5 btn-primary dataset-background'
             }, [
-              'Download selection',
+              'Download Dataset Info',
```
Comment (rushtong) [inline src/pages/DatasetCatalog.js]: Food for thought from my English-minor days: `Info` is a colloquial term that doesn't connote a professional tone, IMHO.

File: src/pages/DatasetCatalog.js [hunk 3]
```diff
@@ -392,23 +393,23 @@ class DatasetCatalog extends Component {
               table({ className: 'table' }, [
                 thead({}, [
                   tr({}, [
+                    th({ isRendered: this.state.isAdmin, className: 'cell-size', style: { textAlign: 'center' } }, ['Actions']),
```
Comment (rushtong) [inline src/pages/DatasetCatalog.js:396]: The actions column is a bit out of alignment. This is from Chrome logged in as admin

Screen Shot 2020-12-14 at 10 11 09 AM

**Review by rushtong — APPROVED** [review]
New table looks good to me, aside from the admin/chair view where there is a bit of a mis-alignment on column size. 
Question:
1. Does this, in fact, `Remove non-ADA compliant font`? It would be nice to know that we've actually addressed this requirement. 
2. Additionally, this only affects the catalog page. To fully address ADA issues, we should create a more comprehensive story and or review of code so we don't leave technical debt behind.

File: src/pages/DatasetCatalog.js (hunk 2 above)
Comment (solideoglori) [reply src/pages/DatasetCatalog.js]: Agreed - changed to "List"

File: src/pages/DatasetCatalog.js (hunk 3 above)
Comment (solideoglori) [reply src/pages/DatasetCatalog.js:396]: This is part of current state. I'm unsure how to resolve and will create a separate ticket.

**Comment by solideoglori** [comment]
I will update the ticket to note that it does update to ADA compliance, per Nancy in UX, but only for the Dataset Catalog page. I will create tickets to update on other specific pages as well.

**Pull request merged**